README.md

```diff
@@ -45,34 +45,34 @@ You can use either a `postgres://` url connection string or the options to defin
-  host                 : '',            // Postgres ip address[s] or domain name[s]
-  port                 : 5432,          // Postgres server port[s]
-  path                 : '',            // unix socket path (usually '/tmp')
-  database             : '',            // Name of database to connect to
-  username             : '',            // Username of database user
-  password             : '',            // Password of database user
-  ssl                  : false,         // true, prefer, require, tls.connect options
-  max                  : 10,            // Max number of connections
-  idle_timeout         : 0,             // Idle connection timeout in seconds
-  connect_timeout      : 30,            // Connect timeout in seconds
-  no_prepare           : false,         // No automatic creation of prepared statements
-  types                : [],            // Array of custom types, see more below
-  onnotice             : fn,            // Defaults to console.log
-  onparameter          : fn,            // (key, value) when server param change
-  debug                : fn,            // Is called with (connection, query, params)
+  host                 : '',                   // Postgres ip address[s] or domain name[s]
+  port                 : 5432,                 // Postgres server port[s]
+  path                 : '',                   // unix socket path (usually '/tmp')
+  database             : '',                   // Name of database to connect to
+  username             : '',                   // Username of database user
+  password             : '',                   // Password of database user
+  ssl                  : false,                // true, prefer, require, tls.connect options
+  max                  : 10,                   // Max number of connections
+  idle_timeout         : 0,                    // Idle connection timeout in seconds
+  connect_timeout      : 30,                   // Connect timeout in seconds
+  no_prepare           : false,                // No automatic creation of prepared statements
+  types                : [],                   // Array of custom types, see more below
+  onnotice             : fn,                   // Defaults to console.log
+  onparameter          : fn,                   // (key, value) when server param change
+  debug                : fn,                   // Is called with (connection, query, params)
   transform            : {
-    column             : fn,            // Transforms incoming column names
-    value              : fn,            // Transforms incoming row values
-    row                : fn             // Transforms entire rows
+    column             : {to?: fn, from?: fn}, // Transforms incoming or outgoing column names
+    value              : {to?: fn, from?: fn}, // Transforms incoming or outgoing row values
+    row                : {to?: fn, from?: fn}, // Transforms entire incoming or outgoing rows
```

Comment:
@porsager wasn't sure of the best way to represent this. Please tweak if have a better idea.